### PR TITLE
Add Adding new guide README instructions

### PR DESCRIPTION
**Pull request by lzap**
SSIA

I also conveted spaces to tabs for the README as majority of it was on tabs.

Also I think that the docinfo.html symlink is actually not necessary, need to test maybe ccutil will require it. So build might fail.

### CODE REVIEW

**Comment by lzap** [comment]
Oh great looks like `docinfo.html` symlinks are not needed, its because `docinfodir` variable overrides it.

**Comment by lzap** [comment]
Wait, docinfo is actually missing from the guides! Damn.

**Comment by lzap** [comment]
False alert, it is there, I double-checked. FYI docinfo is a asciidoctor feature to embed HTML/JS/CSS into guides, we use this for the navigation bar. If you download the build artifact you will not see it but it is there (in the HTML). It only shows when on our real site.

File: guides/README.md [hunk 1]
```diff
@@ -95,6 +95,20 @@ All PDFs should be available for download though.
 
 Please read these guidelines before opening a Pull Request. For more information, see [Guidelines for Red Hat Documentation](https://redhat-documentation.github.io/).
 
+### Creating new guide
```
Comment (maximiliankolb) [inline guides/README.md:98]:
```suggestion
### Creating new guides
```

File: guides/README.md [hunk 2]
```diff
@@ -95,6 +95,20 @@ All PDFs should be available for download though.
...
+Each guide must be in a separate directory in guides/ prefixed with "doc-". The following requirements must be met:
+
+* Top-level file `master.adoc`.
+* Directory `topics` with actual content.
+* Symlink `common` to `../common` for shared content.
+* Directory `images` with images.
+* Symlink `images/common` to `../common/images`.
+* File `Makefile` which includes `../common/Makefile`.
+* File `docinfo.xml` for downstream (Red Hat) guides. Presence of this file will cause build check via `ccutil`, a tool used by Red Hat documentation team.
+
```
Comment (maximiliankolb) [inline guides/README.md:109]: I would appreciate a recommendation to automatically put new content into the `guides/common/modules` directory and stick to one (level one) heading per file.

Everything comes together in either the master.adoc or assemblies, which are lists of files with optional leveloffsets.

**Review by maximiliankolb — APPROVED** [review]
This is a very useful introduction for new contributers! 👍  Maybe you can extend the list as per my comment?

Also: [Youtube: silicon valley: tabs vs spaces](https://www.youtube.com/watch?v=SsoOG6ZeyUI) 😉

File: guides/README.md (hunk 1 above)
Comment (lzap) [reply guides/README.md:98]: Fixed

File: guides/README.md (hunk 2 above)
Comment (lzap) [reply guides/README.md:109]: Done.

File: guides/README.md [hunk 3]
```diff
@@ -95,6 +95,20 @@ All PDFs should be available for download though.
...
+New guides can be typically created as copies of existing guides, just make sure to clean images, topics and `docinfo.xml`.
+
```
[the remark this answers is not in the document]
Comment (lzap) [reply guides/README.md:111]: Oh yes good point, adding.

**Comment by lzap** [comment]
Done, pushed.

**Comment by lzap** [comment]
Pushed.

**Comment by lzap** [comment]
Ping.

**Pull request merged**